src/openrct2/interface/Viewport.cpp

```diff
@@ -1509,7 +1509,7 @@ static bool is_pixel_present_bmp(
     PROFILED_FUNCTION();
 
     // Probably used to check for corruption
```

Comment:
This comment is incorrect. I'm unsure why this check exists but it's not because of corruption. Change the comment to something like "odd, investigate why pure bmp not considered"